sys-devel/llvm/llvm16-16.0.6.recipe

```diff
@@ -602,13 +601,106 @@ INSTALL()
 	# not all arches contain the same libraries. The inventory for each arch
 	# also changes between releases... so lets KISS.
```

Comment:
the comment doesn't match anymore.

Reply:
I understand this comment refers to `libLLVM*` which is still wildcarded... so it's still valid.

Reply:
Ok